src/api/youtubeApi.js

```diff
@@ -0,0 +1,12 @@
+const maxResults = 25;
+
+export async function getVideos(searchItem) {
+  const part = ['snippet', 'id'];
+  const { result } = await window.gapi.client.youtube.search.list({
+    maxResults,
+    part,
+    q: searchItem,
+  });
+
+  return result.items;
+}
```

Comment:
This could have been a custom hook, something with a signature like `const {data, loading, error} = useSearch(searchItem);`